mimoto-issuers-config.json

```diff
@@ -226,6 +226,85 @@
+                  "alt_text": "openg2p-logo"
+              },
+              "title": "Download Beneficiary e-Card",
+              "description": "For OpenG2P Social Cash Transfer Programme.",
+              "language": "en"
+          },
+          {
+              "name": "OpenG2PSocialCashTransferProgram",
+              "logo": {
+                  "url": "https://explore.openg2p.org/g2p_openid_vci/static/description/icon.png",
+                  "alt_text": "openg2p-logo"
+              },
+              "title": "تحميل البطاقة الإلكترونية للمستفيد",
+              "description": "لبرنامج التحويل النقدي الاجتماعي OpenG2P.",
+              "language": "ar"
+          },
+          {
+              "name": "OpenG2PSocialCashTransferProgram",
+              "logo": {
+                  "url": "https://explore.openg2p.org/g2p_openid_vci/static/description/icon.png",
+                  "alt_text": "openg2p-लोगो"
+              },
+              "title": "लाभार्थी ई-कार्ड डाउनलोड करें",
+              "description": "OpenG2P सोशल कैश ट्रांसफर प्रोग्राम के लिए।",
+              "language": "hi"
+          },
+          {
+              "name": "OpenG2PSocialCashTransferProgram",
+              "logo": {
+                  "url": "https://explore.openg2p.org/g2p_openid_vci/static/description/icon.png",
+                  "alt_text": "openg2p-ಲೋಗೋ"
+              },
+              "title": "ಫಲಾನುಭವಿ ಇ-ಕಾರ್ಡ್ ಡೌನ್‌ಲೋಡ್ ಮಾಡಿ",
+              "description": "OpenG2P ಸಾಮಾಜಿಕ ನಗದು ವರ್ಗಾವಣೆ ಕಾರ್ಯಕ್ರಮಕ್ಕಾಗಿ.",
+              "language": "kn"
+          },
+          {
+              "name": "OpenG2PSocialCashTransferProgram",
+              "logo": {
+                  "url": "https://explore.openg2p.org/g2p_openid_vci/static/description/icon.png",
+                  "alt_text": "openg2p-லோகோ"
+              },
+              "title": "பயனாளியின் மின் அட்டையைப் பதிவிறக்கவும்",
+              "description": "OpenG2P சமூக பணப் பரிமாற்றத் திட்டத்திற்கு.",
+              "language": "ta"
+          },
+          {
+              "name": "OpenG2PSocialCashTransferProgram",
+              "logo": {
+                  "url": "https://explore.openg2p.org/g2p_openid_vci/static/description/icon.png",
+                  "alt_text": "open2p-logo"
+              },
+              "title": "I-download ang e-Card ng Benepisyaryo",
+              "description": "Para sa OpenG2P Social Cash Transfer Program.",
+              "language": "fil"
+          }
+      ],
+      "protocol": "OpenId4VCI",
+      "client_id": "${mimoto.oidc.openg2p.partner.clientid}",
+      "client_alias": "openg2p-mimotooidc",
+      "scopes_supported": ["openg2p_beneficiary_1_vc_ldp"],
+      "additional_headers": { "Accept": "application/json" },
+      ".well-known": "https://explore.openg2p.org/api/v1/vci/.well-known/openid-credential-issuer/OpenG2PSocialCashTransferProgram",
+      "redirect_uri": "io.mosip.residentapp.inji://oauthredirect",
```

Comment:
@lalithkota , please add this redirect url "https://injiweb.collab.mosip.net/redirect" also in the client created. so that injiweb can also use the same.

Reply:
@vijay151096 I added the above injiweb URL to the client's redirect uris list. Hence closing this. Let me know if any other changes.